setup.cfg

```diff
@@ -3,7 +3,7 @@ universal = False
 
 [metadata]
 name = cyndilib
-version = 0.0.6
+version = 0.0.7
```

Comment:
Please leave the project version as-is.  I only change that directly before publishing a new release.

The `.. versionadded:: ` tags in docstrings can be left targeting 0.0.7 though